libs/core/src/lib/popover/popover-body/popover-body.component.ts

```diff
@@ -82,7 +75,7 @@ export class PopoverBodyComponent {
     _closeOnEscapeKey = false;
 
     /** Classes added to arrow element */
-    _arrowClasses: string[] = [];
+    _arrowClasses: string;
```

Comment:
Should be private as well as quite a few other properties in this class

Reply:
This property should be public as it's used in template.